tests/tensor/test_math.py

```diff
@@ -1386,6 +1383,12 @@ def test_uint(self):
             n = as_tensor_variable(data)
             assert min(n).dtype == dtype
             i = eval_outputs(min(n))
+            # pytensor.dprint(n)
```

Comment:
The dtype ```uint64``` fails strangely for some reason. The rest work fine. The error message is that ```itype.min = 0``` but ```i``` comes out to be equal to the ```18446744073709551610``` which is the second maximum in the list of ```[0 , 3, 18446744073709551610, 18446744073709551615]```. 

The error is:
```
>           assert i == itype.min
E           assert array(18446744073709551610, dtype=uint64) == 0
E            +  where 0 = iinfo(min=0, max=18446744073709551615, dtype=uint64).min
```

Reply:
Might have to do with the dtype used for internal accumulation.

Reply:
Can you elaborate a little on this?

Reply:
I am not sure what the problem is yet without looking further. But CAReduce has 2 dtypes, the output one and the one used for internal accumulation. I was wondering if the problem was coming from the internal accumulation dtype. Also uint are tricky because they don't represent negative numbers, but I think our implementation of min is something like -max(-x). You may need to investigate a bit the behavior to understand what's going on.